03-talos.tf

```diff
@@ -45,18 +49,32 @@ module "talos_worker_group" {
 resource "talos_machine_secrets" "this" {}
 
 data "talos_machine_configuration" "controlplane" {
+  count = var.controlplane_count
```

Comment:
Could we iterate via the same method (`for_each`) as for the `worker_group` talos_machine_configuration?